src/Agent/NewRelic/Agent/Core/DataTransport/HttpCollectorWire.cs

```diff
@@ -62,9 +62,9 @@ public string SendData(string method, ConnectionInfo connectionInfo, string seri
                 foreach (var header in _requestHeadersMap)
                     request.Headers.Add(header.Key, header.Value);
 
-                using var response = httpClient.SendAsync(request).GetAwaiter().GetResult();
+                using var response = httpClient.SendAsync(request).ConfigureAwait(false).GetAwaiter().GetResult();
```

Comment:
@nrcventura pointed out (from the [doc I linked in the description](https://devblogs.microsoft.com/dotnet/configureawait-faq/#when-should-i-use-configureawaitfalse#im-using-getawaiter-getresult-do-i-need-to-use-configureawaitfalse))  that `.ConfigureAwait(false).GetAwaiter().GetResult()` does exactly nothing different than just `.GetAwaiter().GetResult()` -- and that was the one area I was *really* hoping we'd be able to gain some improvement around. 

I will note, however, that the `.SendAsync()` call here is actually calling into `NRHttpClient.SendAsync()` where the added `.ConfigureAwait(false)` is actually useful. So, while this change may not do anything beneficial here, it also does no harm.